Python/Product/PythonTools/PythonTools.csproj

```diff
@@ -719,7 +720,7 @@
     <None Include="Theme\dark.vstheme" />
     <None Include="Theme\make_theme_pkgdef.py" />
     <Content Include="$(PackagesPath)\Newtonsoft.Json\lib\net45\Newtonsoft.Json.dll">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Do we really need to copy always? This causes builds to fail if the DLL is still in use (which happened almost immediately to me for some reason)